core-kotlin/src/main/java/com/amplifyframework/kotlin/auth/Auth.kt

```diff
@@ -184,10 +185,12 @@ interface Auth {
      * to that plugin which contains the various security tokens and other identifying information if you want to
      * manually use them outside the plugin. Within Amplify this should not be needed as the other categories will
      * automatically work as long as you are signed in.
+     * * @param options Advanced options for fetching auth session.
+     *                If not provided, default options will be used.
      * @return Information about the current authenticated session, where applicable
      */
     @Throws(AuthException::class)
-    suspend fun fetchAuthSession(): AuthSession
+    suspend fun fetchAuthSession(options: AuthFetchSessionOptions): AuthSession
```

Comment:
options should not be mandatory. Please add override with options.

Reply:
Thanks, missed this.